IAM/Frequently_asked_questions.mediawiki

```diff
@@ -0,0 +1,49 @@
+=== Mozilla IAM FAQ (Frequently Asked Questions) ===
+
+==== '''Q''': ''What is Mozilla IAM?'' ====
+
+Mozilla IAM stands for Mozilla's Identity and Access Management. It's the internal login system that Mozilla uses to login to various web properties and systems.
+
+Usually, you'd use Mozilla IAM as a Mozilla Employee, or has a contributor with access to the tools and resources Mozilla uses day to day.
+An example of that would be Air Mozilla: https://air.mozilla.org
+
+==== '''Q''': ''How do I login with Mozilla IAM?'' ====
+
+Mozilla IAM supports various login methods, such as "LDAP" (Employee logins), GitHub social login, Google social login and email login (which we call "passwordless").
```

Comment:
Should we say "Staff logins" instead of "Employee logins"?

Reply:
replaced all instances of employe* by staff